examples/full-example/app.js

```diff
@@ -47,4 +47,27 @@ function bundle(req, bundle, variations) {
     return '<script src="'+url+'"></script>';
 }
 
+function entryMap(req, bundle, variations) {
+    // req.mendel.getBundleEntries contains all bundles as keys and array of
+    // entries that were used and normalized by variations, this allows apps
+    // to do their specific logic with their bundles
+    var bundles = req.mendel.getBundleEntries();
+
+    // In this particular case, we used "temporary" on the .mendelrc
+    // go expose our lazy bundle entries from extractify
+    var lazy =  [
+        '<script>',
+        '   (function(){',
+        '       var nameSpace = "_extractedModuleBundleMap";',
+        '       var url = "'+req.mendel.getURL(bundle, variations)+'";',
```

Comment:
As explained, yes, we need one `findTreeForVariations` per bundle that is **used** per request. But looping through all available bundles does not mean used. Also, by letting it be granular we allow people to flush the essential URLs first and eventually flush some more if they still need, making it an API that encourages performance improvements, while #34 removes this option and encourages duplicate calls to `findTreeForVariations`.